hw/display/virtio-dmabuf.c

```diff
@@ -13,60 +13,76 @@
 #include "hw/virtio/virtio-dmabuf.h"
 
 
-static struct shared_resources {
-    GHashTable *uuids;
-    bool locked;
-} res = {
-    .uuids = NULL,
-    .locked = false,
-};
+#define UUID_TO_POINTER(i) \
+    (gpointer) (g_intern_static_string(qemu_uuid_unparse_strdup((&i))))
```

Comment:
AFAIK there is a good practice to wrap with parentheses all macro parameters and the macro itself, to protect from some unexpected usages:
```
((gpointer) (g_intern_static_string(qemu_uuid_unparse_strdup((&(i))))))
```